### PR TITLE
br-4528

### PR DESCRIPTION
**Pull request by pallavipannu**

### CODE REVIEW

File: tensorflow/classification/resnet/catsdogs/README.md [hunk 1]
```diff
@@ -17,83 +17,92 @@ This example is derived from [retraining](https://www.tensorflow.org/hub/tutoria
...
+
+## How to Serve
+After the job is complete from above step. The trained model will get generated inside Dkube.
+
+1. Click on the training run named *run-training-Catsdogs.
+2. Select Lineage tab beside the Summary details page.
+3. Click on OUTPUTS model Catsdogs
+4. Click on the Test Inference button.
+5. Input the unique name say catsdogs-serving
+6. Select CPU or GPU to deploy the model on a specific device. Unless specifically required, model can be served on CPU
```
Comment (AnsuFrancis) [inline tensorflow/classification/resnet/catsdogs/README.md:94]: @pallavi-pannu-oc We need to provide details of serving image, trnasformer image, transformer prject and code here right?

File: tensorflow/classification/mnist/digits/README.md [hunk 2]
```diff
@@ -17,84 +17,96 @@ This example is derived from [tensorflow example](https://github.com/tensorflow/
...
- 8. Wait till *status* field shows *running*.
- 9. Copy the *URL* shown in *Endpoint* field of the serving job.
+ 1. After the job is complete from above step. The trained model will get generated inside Dkube.
+ 2. Click on the training run named *run-training-mnist-digits.
+ 3. Select Lineage tab beside the Summary details page.
+ 4. Click on OUTPUTS model mnist-digits
+ 5. Click on the Test Inference button.
+ 6. Input the unique name say mnist-serving
+ 7. Select CPU or GPU to deploy the model on a specific device. Unless specifically required, model can be served on CPU
+ 8. Click on Test inference button
```
Comment (AnsuFrancis) [inline tensorflow/classification/mnist/digits/README.md:97]: @pallavi-pannu-oc Please provide details of serving image, transformer image, transformer project, transformer code etc here

File: tensorflow/classification/mnist/digits/README.md [hunk 3]
```diff
@@ -17,84 +17,96 @@ This example is derived from [tensorflow example](https://github.com/tensorflow/
...
- 1. Click *Jobs* side menu option.
- 2. Click *+Training Job* button.
+1. Click Repos side menu option.
+2. Click +Datasets button under Datasets section.
+3. Enter a unique name say mnist-digits
+4. Enter tag name in Tag text-box and field is optional
+5. Select Versioning as DVS 
+6. Select Dataset store as default
+7. Select Dataset Source as Git
+8. Paste link https://github.com/oneconvergence/dkube-examples/tree/2.0.7/tensorflow/classification/mnist/digits/classifier/data in the URL text box
```
Comment (AnsuFrancis) [inline tensorflow/classification/mnist/digits/README.md]: @pallavi-pannu-oc Branch is 2.0.7?

File: tensorflow/classification/mnist/digits/README.md [hunk 4]
```diff
@@ -17,84 +17,96 @@ This example is derived from [tensorflow example](https://github.com/tensorflow/
...
- 7. Workspace will be created and imported in Dkube. Progress of import can be seen.
- 8. Please wait till status turns to *ready*.
+## Step1: Create a Project
+
+1. Click Repos side menu option.
+2. Click +Project button under Projects section.
+3. Enter a name say mnist-digits
+4. Enter tag name in Tag text-box
+5. Select Project Source as Git
+6. Paste link https://github.com/oneconvergence/dkube-examples/tree/2.0.7/tensorflow/classification/mnist/digits/classifier/program in the URL text box.
```
Comment (AnsuFrancis) [inline tensorflow/classification/mnist/digits/README.md]: @pallavi-pannu-oc Branch 2.0.7?

File: tensorflow/classification/resnet/catsdogs/README.md (hunk 1 above)
Comment (pallavipannu) [reply tensorflow/classification/resnet/catsdogs/README.md:94]: This doc i used https://docs.google.com/document/d/1aezz4OeCsBIejHr1xXioZM1AlhbCPeXNvL-kI3fbby8/edit

**Pull request merged**